dotnet-desktop-guide/framework/winforms/controls/bindingnavigator-control-overview-windows-forms.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "BindingNavigator Control Overview"
+description: Learn more about and overview of the BindingNavigator control in WIndows Forms, which can be used to create a standardized means for users to search and change data.
```

Comment:
Typo, should be "Windows Forms" instead of "WIndows Forms"

Reply:
Addressed.